src/main/resources/config/application.properties

```diff
@@ -122,15 +122,16 @@ feature.postponements.enabled: ${POSTPONEMENTS_FEATURE:false}
 feature.sscs2.enabled: ${SSCS2_FEATURE:false}
 feature.work-allocation.enabled: ${WORK_ALLOCATION_FEATURE:false}
 feature.gaps-switchover.enabled: ${GAPS_SWITCHOVER_FEATURE:false}
-feature.snl.enabled: ${SCHEDULE_LISTING_FEATURE:false}
+feature.snl.enabled: ${SCHEDULE_LISTING_FEATURE:true}
```

Comment:
Not quite released yet

Reply:
Woops, its a different feature flag for this anyway :)